ui/app/templates/components/mfa-setup-step-one.hbs

```diff
@@ -1,26 +1,37 @@
-<p>
-  TOTP Multi-factor authentication (MFA) can be enabled here if it is required by your administrator. This will ensure that
-  you are not prevented from logging into Vault in the future, once MFA is fully enforced.
-</p>
-<form id="mfa-setup-step-one" {{on "submit" this.verifyUUID}}>
-  <MessageError @errorMessage={{this.error}} class="has-top-margin-s" />
-  <div class="field has-top-margin-l">
-    <label class="is-label">
-      Method ID
-    </label>
+<div ...attributes>
```

Comment:
only thing I did to this file was add some data-test selectors and the div here to wrap the hbs in an div with attributes